alerts/kubelet.libsonnet

```diff
@@ -47,6 +47,32 @@
               message: "Kubelet '{{ $labels.node }}' is running at {{ $value | humanizePercentage }} of its Pod capacity.",
             },
           },
+          {
+            alert: 'KubeNodeReadinessFlapping',
+            expr: |||
+              sum(changes(kube_node_status_condition{status="true",condition="Ready"}[15m])) by (node) > 2
+            ||| % $._config,
+            'for': '15m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'The readiness status of node {{ $labels.node }} has changed {{ $value }} times in the last 15 minutes.',
+            },
+          },
+          {
+            alert: 'KubeletPlegDurationHigh',
+            expr: |||
+              histogram_quantile(0.99, sum(rate(kubelet_pleg_relist_duration_seconds_bucket[5m])) by (node, le)) >= 10
```

Comment:
We unfortunately can't rely on the node label being available here as that's not a label on the metric but added via relabeling, but we can't enforce users to have a specific configuration. We need to do a join similar to what we do [here](https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/bdf6d50253b81f9262b4cbf47f21cab7d2cc9141/alerts/kubelet.libsonnet#L40), with the `kubelet_node_name` metric that the kubelet exposes.